tests/system/system_test.go

```diff
@@ -22,15 +23,22 @@ func NewSystemTest(t *testing.T, dir string, vars map[string]string) *SystemTest
 	vars["endpoint"] = os.Getenv(config.ENVVAR_ENDPOINT)
 	vars["token"] = os.Getenv(config.ENVVAR_TOKEN)
 
+	fmt.Println("[DEBUG] endpoint", vars["endpoint"])
+	fmt.Println("[DEBUG] token", vars["token"])
+
 	tfContext := tftest.NewTestContext(t,
 		tftest.Dir(dir),
 		tftest.Vars(vars),
 		tftest.DryRun(*dry))
 
 	layer0 := clients.NewLayer0TestClient(t, vars["endpoint"], vars["token"])
 
-	// download modules using terraform get
-	tfContext.Terraformf("get")
+	// download modules using terraform init
+	stdoutStderr, err := tfContext.Terraformf("init")
+	if err != nil {
+		fmt.Println("[ERROR]", err)
+	}
+	fmt.Printf("%s\n", stdoutStderr)
```

Comment:
seems like the printing I'm seeing here is leftover for live debugging more than actual code intended to stay?

Reply:
Was for debugging, but switched to using t.Logf

Reply:
see my `t.Logf` vs `log.Printf("[DEBUG] ...` above